src/CopilotTag.php

```diff
@@ -1,6 +1,20 @@
 <?php
 namespace CopilotTags;
 
+
+// TODO: Put these somewhere sensible
+const blockRegex = "/\[#\w+:\s*\S+?(?:\s?(?:\w+:\s?\w+)*?)\](?:\((?:\d+(?:%|em|ex|px|pt|pc|cm|mm|in)?)(?:x?(?:\d+(?:%|em|ex|px|pt|pc|cm|mm|in)?)*)\))*(?:\|{3}[\S\s]*?\|{3})*/";
+
+function getOuterSpace($text = "") {
```

Comment:
(I could be wrong about the global scope thing, maybe it's fine because it's namespaced... I still think it should be a static function on a class).

Reply:
> I'm wondering why this is named similarly to the outerspace library when the functionality is quite different.

Just because it gets the outer space. Wasn't meant to be similar.

> This needs to be a static function on a class. Otherwise it is probably muddying the global scope as a side-effect of autoloading, which a class library definitely should not be doing.

Cool, was wondering what to do with it, as per the comment on line 5.

Reply:
@graycodes oh I see, I didn't see that was referring to both!

Reply:
It's cool! Wanted to know where was sensible to put it!